proto/p2p/p2p.proto

```diff
@@ -61,297 +61,361 @@ message Message {
+// Upon reception of a Ping, peers should respond with a Pong message.
 message Ping {
-  // uptime is the primary network uptime percentage.
+  // Uptime percentage on the primary network [0, 100]
   uint32 uptime = 1;
-  // subnet_uptimes contains subnet uptime percentages.
+  // Uptime percentage on subnets
   repeated SubnetUptime subnet_uptimes = 2;
 }
 
-// Contains subnet id and the related observed subnet uptime of the message
-// receiver (remote peer).
+// SubnetUptime is a descriptor for a peer's uptime on a subnet.
 message SubnetUptime {
+  // Subnet the peer is validating
   bytes subnet_id = 1;
+  // Uptime on the subnet [0, 100]
   uint32 uptime = 2;
 }
 
-// Contains the uptime percentage of the message receiver (remote peer)
-// from the sender's point of view, in response to "ping" message.
-// Uptimes are expected to be provided as integers ranging in [0, 100].
+// Pong is sent in response to a Ping with the perceived uptime of the
+// peer.
 message Pong {
-  // Deprecated: remove all these fields in the future, but keep the message.
-  // uptime is the primary network uptime percentage.
+  // Deprecated: uptime is now sent in Ping
+  // Uptime percentage on the primary network [0, 100]
   uint32 uptime = 1;
-  // subnet_uptimes contains subnet uptime percentages.
+  // Deprecated: uptime is now sent in Ping
+  // Uptime percentage on subnets
   repeated SubnetUptime subnet_uptimes = 2;
 }
 
-// The first outbound message that the local node sends to its remote peer
-// when the connection is established. In order for the local node to be
-// tracked as a valid peer by the remote peer, the fields must be valid.
-// For instance, the network ID must be matched and timestamp should be in-sync.
-// Otherwise, the remote peer closes the connection.
-// ref. "avalanchego/network/peer#handleVersion"
-// ref. https://pkg.go.dev/github.com/ava-labs/avalanchego/network#Network "Dispatch"
+// Version is the first outbound message sent to a peer when a connection is
+// established to start the p2p handshake.
+//
+// Peers must respond to a Version message with a PeerList message to allow the
+// peer to connect to other peers in the network.
+//
+// Peers will close a connection with a peer that sends it an incompatible
+// Version message.
 message Version {
+  // Network the peer is running on (e.g local, testnet, mainnet)
   uint32 network_id = 1;
+  // Unix timestamp when this Version message was created
   uint64 my_time = 2;
+  // IP address of the peer
   bytes ip_addr = 3;
+  // IP port of the peer
   uint32 ip_port = 4;
+  // Avalanche client version
   string my_version = 5;
+  // Timestamp of the IP
   uint64 my_version_time = 6;
+  // Signature of the peer IP port pair at a provided timestamp
   bytes sig = 7;
+  // Subnets the peer is tracking
   repeated bytes tracked_subnets = 8;
 }
 
-// ref. https://pkg.go.dev/github.com/ava-labs/avalanchego/utils/ips#ClaimedIPPort
```

Comment:
I generally got rid of these code references in this file to avoid it from going out-of-sync w/ any refactors we do